docs/README-freebsd.md

```diff
@@ -44,7 +61,13 @@ BROKEN (build crashes on FreeBSD 11.2)
 ```
 
 ```
+cd web3.py
+tox
```

Comment:
Maybe `tox -e py36-core` and others, since it's unreasonable to run the whole suite at once right now.

Reply:
Well, right... I've just followed "It is important to understand that each Pull Request must pass the full test suite as part of the CI check, hence it is often convenient to have tox installed locally as well." rule from https://py-evm.readthedocs.io/en/latest/contributing.html. So do you want me to update that?

Reply:
To clarify, I'm suggesting something like:
```
tox -e py36-core
tox -e py36-ens
etc
```

Hm, I don't disagree with anything in that statement (also it's in py-evm docs instead of web3.py). How are you suggesting the text should be updated?

Reply:
I've messes up py-evm with web3.py again, sorry... Yes, your suggestion looks ok, I'll update pull request.